services/apps/alcs/src/portal/notification-submission/notification-submission.service.ts

```diff
@@ -406,18 +406,19 @@ export class NotificationSubmissionService {
     submission: NotificationSubmission,
     document: NotificationDocument,
     user: User,
-  ) {
+  ): Promise<boolean> {
     const templateData = await this.generateSrwEmailData(submission, document);
 
-    const didSend = await this.emailService.sendEmail({
-      to: [templateData.to],
-      body: templateData.html,
-      subject: `Agricultural Land Commission SRW${submission.fileNumber} (${submission.applicant})`,
-      parentType: PARENT_TYPE.NOTIFICATION,
-      parentId: templateData.parentId,
-      cc: templateData.cc,
-      attachments: [document.document],
-    });
+    const didSend =
+      (await this.emailService.sendEmail({
+        to: [templateData.to],
+        body: templateData.html,
+        subject: `Agricultural Land Commission SRW${submission.fileNumber} (${submission.applicant})`,
+        parentType: PARENT_TYPE.NOTIFICATION,
+        parentId: templateData.parentId,
+        cc: templateData.cc,
+        attachments: [document.document],
+      })) ?? false;
```

Comment:
Rather than use nullish coalescing, you can fix the email service by changing return; to return false; on line 137 of email.service.ts.